### PR TITLE
Fix new unreachables/dumb code

### PR DESCRIPTION
**Pull request by spookydonut**
## About The Pull Request
scream

### CODE REVIEW

File: code/modules/antagonists/abductor/equipment/abduction_gear.dm [hunk 1]
```diff
@@ -518,7 +518,7 @@ Congratulations! You are now trained for invasive xenobiology research!"}
 		toggle_on(user)
 
 	if(iscyborg(target))
-		if(BATON_STUN)
```
Comment (Qustinnus) [inline code/modules/antagonists/abductor/equipment/abduction_gear.dm]: lol how do u even find these

**Comment by trollbreeder** [comment]
spookydonut you're the unsung hero of tgstation code

File: code/modules/antagonists/abductor/equipment/abduction_gear.dm (hunk 1 above)
Comment (Jalleo) [reply code/modules/antagonists/abductor/equipment/abduction_gear.dm]: Its due a new feature that is referenced in one of these other PR's or one with updating spacemandmm or something else I think it was.

**Comment by spookydonut** [comment]
merged into #49712

**Pull request closed without merging**